Description: Enable svm by default for qemu64 cpu on AMD
 Upstream changed the default behavior late in 2014 with the patch:
 .
 75d373ef97: target-i386: Disable SVM by default in KVM mode.
 .
 We enable vmx by default for intel, so continue to enable svm by default
 as well.
 .
 This is not appropriate for upstreaming, as enabling svm is a distro
 decision.
Author: Stefan Bader <stefan.bader@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/libvirt/+bug/1561019
Forwarded: no

Index: qemu/hw/i386/pc_piix.c
===================================================================
--- qemu.orig/hw/i386/pc_piix.c
+++ qemu/hw/i386/pc_piix.c
@@ -308,7 +308,6 @@ static void pc_compat_2_2(MachineState *
 static void pc_compat_2_1(MachineState *machine)
 {
     pc_compat_2_2(machine);
-    x86_cpu_change_kvm_default("svm", NULL);
 }
 
 static void pc_compat_2_0(MachineState *machine)
Index: qemu/target/i386/cpu.c
===================================================================
--- qemu.orig/target/i386/cpu.c
+++ qemu/target/i386/cpu.c
@@ -1403,7 +1403,6 @@ static PropValue kvm_default_props[] = {
     { "x2apic", "on" },
     { "acpi", "off" },
     { "monitor", "off" },
-    { "svm", "off" },
     { NULL, NULL },
 };
 
